client/dfclient/src/death_report_processor.cpp

```diff
@@ -0,0 +1,433 @@
+#include "death_report_processor.hpp"
+
+#include "../../../common/constants.hpp"
+
+#include "game_data.hpp"
+#include "gameplay_state.hpp"
+
+DrawableNodeVector DeathReportProcessor::ProcessDeathReport(const FlatBuffGenerated::DeathReport* deathReport) {
+	
+	if (deathReport->killer() == gameData.myPlayerID) {
+		// i killed someone
+		return processKill(deathReport);
+		
+	} else if (deathReport->victim() == gameData.myPlayerID) {
+		// i died :c
+		return processDeath(deathReport);
+	} else {
+		// somebody died
+		return processObituary(deathReport);
+	}
+}
+
+/* ----------- I KILLED SOMEBODY ------------ */
+
+DrawableNodeVector DeathReportProcessor::processKill(const FlatBuffGenerated::DeathReport* deathReport) {
+		_resources.playSound(SoundType::KILL);
```

Comment:
This method is indented too much. Every line is indented with at least 2 tabs - please fix it.